packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -262,7 +262,7 @@
             }
 
             @include desktop {
-                grid-template-columns: repeat(4, 1fr);
+                grid-template-columns: 1fr 1fr 1fr 1.5fr;
```

Comment:
@shafin-deriv maybe could you please try `grid-template-columns: repeat(3, 1fr), 1.5fr; `?